Lib/tarfile.py

```diff
@@ -2272,6 +2272,13 @@ def makelink(self, tarinfo, targetpath):
           (platform limitation), we try to make a copy of the referenced file
           instead of a link.
         """
+        # do not create link if target path is equals to link target
+        if (
```

Comment:
could you use the PEP8 style for the condition? 
for example
```python
if hasattr(tarinfo, "_link_target") and tarinfo._link_target == targetpath:
```

Reply:
The line is 83 characters long if I do not split it into two lines.

Reply:
The preferred way to split it is as follows:
```
if hasattr(tarinfo, "_link_target") and \
    tarinfo._link_target == targetpath:
```